api/src/services/stardust/stardustApiService.ts

```diff
@@ -249,18 +249,21 @@ export class StardustApiService {
     public async basicOutputDetailsByAddress(addressBech32: string): Promise<IAddressDetailsResponse> {
```

Comment:
Do we use this for fetches outside `associatedOutput` stuff ? I would prefer we introduce a separate method with the filtering maybe, and leave this "all outputs" untouched. 🤔

Reply:
In `useAddressBasicOutputs` hook we use `client.basicOutputsDetails`, which calls api `basicOutputDetailsByAddress`, to get basic outputs for AddressState so I think we are good. If we introduce new method than this one wouldn't be used.